contracts/src/oracle/library.cairo

```diff
@@ -175,6 +179,13 @@ namespace Oracle {
         return (price, decimals, last_updated_timestamp, entries_len);
     }
 
+    func get_value{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(key: felt) -> (
+        value: felt, decimals: felt, last_updated_timestamp: felt, num_sources_aggregated: felt
+    ) {
+        let (entry_) = Oracle__generic_entry_storage.read(key);
```

Comment:
how do we want to handle this?  We can standardize entries with 18 decimal points.

Reply:
Yup, sounds good. At some point we may want to move this back to having a variable number of decimals with a separate `decimals_storage(key) -> (decimals)` but given this API that should be backward compatible and no current use cases need that flexibility

Reply:
Ugh, think we need to fix `get_decimals` though...
To something like
```
get_generic_decimals(key) -> (decimals)
get_spot_decimals(pair_id) -> (decimals)
get_future_decimals(pair_id, expiry_timestamp) -> (decimals)
```

Reply:
do we need get_generic_decimals if it's always 18?

Reply:
I'd say this probably isn't within the scope of this PR and we can address it in a separate PR